### PR TITLE
editorconfig check: instruct user what to do

### PR DESCRIPTION
**Pull request by domenkozar**

### CODE REVIEW

File: .github/workflows/editorconfig.yml [hunk 1]
```diff
@@ -39,3 +39,8 @@ jobs:
       if: env.PR_DIFF
       run: |
         echo "$PR_DIFF" | xargs editorconfig-checker -disable-indent-size
+    - if: ${{ failure() }}
+      run: |
+        echo "::error :: Hey! It looks like your changes do not respect editorconfig settings. Read https://editorconfig.org/#download to configure your editor."
```
Comment (grahamc) [inline .github/workflows/editorconfig.yml:44]:
```suggestion
        echo "::error :: Hey! It looks like your changes don't follow our editorconfig settings. Read https://editorconfig.org/#download to configure your editor so you never see this error again."
```

maybe?

File: .github/workflows/editorconfig.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name: "Checking EditorConfig"
 permissions: read-all
 
 on:
-  pull_request_target:
+  pull_request:
```
Comment (zowoq) [inline .github/workflows/editorconfig.yml:6]: Why revert the change that we just merged in https://github.com/NixOS/nixpkgs/pull/120959?

Comment (domenkozar) [reply .github/workflows/editorconfig.yml:6]: I don't think it really needs _target? User github token should be enough to fetch the pull request metadata.

Comment (zowoq) [reply .github/workflows/editorconfig.yml:6]: It isn't for the metadata, it's to avoid need to manually approve the action.

Comment (zowoq) [reply .github/workflows/editorconfig.yml:6]: https://github.blog/2021-04-22-github-actions-update-helping-maintainers-combat-bad-actors/

**Comment by zowoq** [comment]
https://github.com/zowoq/actions-test/pull/1

Opening the log from the PR it looks like the message isn't being displayed properly, it's blocked by check error?

**Comment by domenkozar** [comment]
See https://github.com/zowoq/actions-test/actions/runs/795424920

**Comment by zowoq** [comment]
https://github.com/zowoq/actions-test/pull/1/checks?check_run_id=2465001774

Screenshot 2021-04-29 at 22-22-25 Update README md by zowoq · Pull Request #1 · zowoq actions-test

**Comment by zowoq** [comment]
If you're happy with how it looks I don't have an objection. Feel free to merge.

**Comment by domenkozar** [comment]
I'm not happy, but there's no better way without writing code to set status, which would require write permission token for all contributions.

This is the best we can do about it.

**Comment by rhendric** [comment]
The message chosen here (‘Hey! It looks like your changes don't follow our editorconfig settings. Read https\://editorconfig.org/#download to configure your editor so you never see this error again.’) is under consideration for revision in #287487, if any participants have a preference for the current wording.
